core/src/main/java/dk/alexandra/fresco/logging/PerformanceLogger.java

```diff
@@ -17,7 +19,7 @@
   public DecimalFormat df = new DecimalFormat("#.00");
   
   public enum Flag {
-    LOG_NETWORK, LOG_RUNTIME, LOG_NATIVE_BATCH;
+    LOG_NETWORK, LOG_EVALUATOR, LOG_NATIVE_BATCH;
```

Comment:
Maybe we should consider removing this enum for good? It doesn't cover all loggers anyways, and no logger decorators are created by the framework itself anyways. You have to do this yourself. Vote to remove. It used to make sense when decorators was created within the framework.

Reply:
How was it used before?

Reply:
It is/was used to control if a certain decorator should be used when initializing the sce. It's only really used in the AbstractDummy*Tests